app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -260,7 +260,7 @@ abstract class BaseBrowserFragment : Fragment(), BackHandler, SessionManager.Obs
                         DownloadService::class
                     ),
                     onNeedToRequestPermissions = { permissions ->
-                        requestPermissions(permissions, REQUEST_CODE_DOWNLOAD_PERMISSIONS)
+                        requestPermissions(permissions, R.integer.REQUEST_CODE_DOWNLOAD_PERMISSIONS)
```

Comment:
Please create `R.id` resources instead of `R.integer` resources. Also use a lowercase name for the resources (ie: request_code_download_permissions).

Reply:
I have made the changes